CODE-OF-CONDUCT.md

```diff
@@ -34,6 +34,78 @@ In keeping with this commitment, we offer the following guidelines:
+  more details refer to the [CNCF Code of Conduct][cncf-coc]).
+- If you are uncomfortable with a piece of content (but it may not necessarily
+  violate the code of conduct), we suggest sending a private message to the
+  content owner expressing your concerns. If this is not resolved, you may wish
+  to request the help of a Co-Chair/TL via cncf-tag-security-leads@lists.cncf.io
+  to help mediate the situation.
+- Discussions about these issues are important, and there are great avenues to
+  discuss them. This includes bringing up concerns to the [CNCF TOC][cncf-toc]
+  (which can be done through discussion with Security TAG leadership) or talking
+  to Security TAG leadership about moderating a post. To help ensure that we can
+  give focus to these issues and not tangle them up with technical discussions,
+  we should keep these discussions separate from channels which are focused
+  around technical exchange.
+
+For content creators:
+
+- Content must strive to remain _on-topic_, particularly where video and images
+  are provided.  Use of emojis and gifs as responses are content in and of
+  themselves need to be relevant to the particular post. For examples please
+  refer to the reference section below. 
+- If you receive a notice about a piece of content you've created, please seek
+  to understand that in some cases you may not agree with a decision or request.
+  Being able to practice tolerance and mindfulness is just as important to keep
+  the community working towards a common goal. The mediation and resolution
+  system that we have in place aims to handle this with the hope that both
+  content creators and consumers are heard and represented. These situations are
+  not zero sum, and often we aim to reach an agreeable compromise where a
+  discussion of a topic can happen without making members of the community feel
+  uncomfortable.
+- In the event where there is disagreement, we have some guidelines that can
+  help prevent escalation
+  - Do not bring the discussion out of context.
+  - Do not rationalize the actions you take. We do not expect anyone to
+    understand what everyone else feels towards certain things (e.g. the same
+    gestures in certain cultures are good and bad in others). Understand that
+    something may not be wrong, but it may affect others.
+
+In summary, be nice, inclusive and welcoming. Misunderstandings, mistakes and
+oversights happen, and when they do, there are some good ways to go about having
+a conversation with colleagues to make our community inclusive and welcoming to
+everyone!  
+
+## Reference
+
+Example of reasonable gif: Group is close to wrapping up deliverable, as part of
+an update, the lead posts a "nearly done" gif.  
+
+Example of reasonable emoji:  Post in the group uses emojis to break up content
+and is relevant to the item discussed or used in response to post to signify
+voting, opinion, acceptance, emotion, etc.  
```

Comment:
I'm not sure if i know how to describe it. I don't think ive personally seen a case of unacceptable emojis being used, but im not a big GIF/emoji user, so not too familiar with what it would constitute.. i do think stating what's ok would be useful enough for folks to be able to vet their own content.

Reply:
@ashutosh-narkar i think for now this is sufficient.  If/when an unacceptable gif or emoji comes up, we can enumerate it here.

Reply:
Sounds good.